tests/lib/rules/no-unknown-property.js

```diff
@@ -30,6 +30,8 @@ ruleTester.run('no-unknown-property', rule, {
     {code: '<App class="bar" />;'},
     {code: '<App for="bar" />;'},
     {code: '<App accept-charset="bar" />;'},
+    {code: '<meta charset="utf-8" />;'},
+    {code: '<meta charSet="utf-8" />;'},
```

Comment:
i think we should be forcing `charset`; HTML also supports `CHARSET` and `cHaRsEt`, but lowercase is the very dominant convention.

Reply:
That was my initial thought, but `charSet` is not an unknown attribute, as the name of the rule implies. That said, I’ll modify the PR, so you decide. 😉

Reply:
hmm, that is true. I suppose it’s a slight overreach of this rule to force it.

Reply:
Yeah. As much as I would like for people to write `charset` instead of `charSet`, raising an error because of that would not be symmetrical with raising an error for `acceptcharset` in favor of `acceptCharset`, because only the latter would result into the correct form of the attribute – `accept-charset`.

I'll hold with the changes then until we agree what they should be.

Reply:
no need, i think this makes sense as-is